Documentation/operations/upgrade.rst

```diff
@@ -305,7 +305,9 @@ communicating via the proxy must reconnect to re-establish connections.
   on your cluster, the number of identities will temporarily increase during the upgrade, which will result in increased drops. If you would like 
   to disable this new behavior, you can add ``!io\.cilium\.k8s\.policy\.serviceaccount`` to your identity-relevant labels to 
   exclude the ``io.cilium.k8s.policy.serviceaccount`` label.
-
+* The Helm value of ``enableIPv4Masquerade`` in ``eni`` mode will change from ``true`` to ``false`` by default from 1.18.
```

Comment:
Not sure if we discuss `upgradeCompatability` in upgrade notes? Either way, this should not affect existing clusters.

Reply:
I can remove it if we dont discuss about it